src/Uno.UI.Runtime.Skia.Android/Microsoft/UI/Xaml/UnoSKCanvasView.cs

```diff
@@ -99,7 +120,15 @@ protected override void OnDraw(Canvas c)
 				new SKPaint { Color = SKColors.Red});
 #endif
 
-			_picture = recorder.EndRecording();
+			var picture = recorder.EndRecording();
+
+			Interlocked.Exchange(ref _picture, picture);
+
+			if (this.Log().IsEnabled(LogLevel.Trace))
+			{
+				this.Log().Trace("Resume drawing");
+			}
+			RenderMode = Rendermode.Continuously;
```

Comment:
Why do we need to set this here in addition to OnCompositionRenderingActiveChanged

Reply:
We need this because of manual triggers of rendering, mouse moves, pointers, etc...